json.cpp

```diff
@@ -0,0 +1,68 @@
+#include "json.h"
+#include <sstream>
+
+std::map<std::string, std::string> Parser::Parse_string(std::string fileName)
+{
+   
+    std::map<std::string, std::string> strg;
+	std::string ln,ln1,ln2;
+	std::istringstream stream(fileName);
+
+	while (stream)
+    {
+		ln1 = "", ln2 = "";
+		std::getline(stream, ln);
+		if (ln.find('{') == std::string::npos && ln.find('}') == std::string::npos)
+        {
+			for (unsigned int i = 0; i < ln.find(':'); i++)
```

Comment:
Tehát minden karakter megvizsgálása után megkeresteted a : helyét? Nem lesz túl gyors... Arról nem beszélve, hogy valaki egy gépelési hiba folytán el is feledkezhet erről a jelről, és az ilyen hibákra jó lenne készülni.